website/integrations/services/argocd/index.md

```diff
@@ -69,16 +132,25 @@ In the `argocd-secret` Secret, add the following value to the `data` field:
 dex.authentik.clientSecret: <base 64 encoded value of the Client Secret from the Provider above>
 ```
 
+If using helm, the above can be added to `configs.secret.extra` like so, securely substituting the string however you see fit:
```

Comment:
```suggestion
If using Helm, the above can be added to `configs.secret.extra` as shown below, securely substituting the string however you see fit:
```

Reply:
I'd like to further edit this, to identify/qualify and be very specific on what "the above <?>" is... Can we say 

"If using Helm, the above string/line/definition/? can be added to a  `configs.secret.extra` section in the <__??__> file, as shown below, securely substituting the string however you see fit:"

These are nits (well not the capitalization, that's important), but for new users, it's always nice to be a bit pedantic, and say where (in what file) we are talking about.

Reply:
Ammended, see what you think!